metatomic-torch/src/register.cpp

```diff
@@ -219,7 +219,32 @@ TORCH_LIBRARY(metatomic, m) {
 
     // standalone functions
     m.def("version() -> str", version);
-    m.def("pick_device(str[] model_devices, str? requested_device = None) -> str", pick_device);
+    // Expose pick_device to Python. The C++ helper returns a c10::DeviceType;
+    // build a torch::Device from it (with default index policy) and return its
+    // string representation to Python (backwards-compatible).
+    m.def(
+        "pick_device(str[] model_devices, str? requested_device = None) -> str",
+        [](const std::vector<std::string> &model_devices,
```

Comment:
can you move this code in a standalone function on top of the file, to keep the bindings more readable?